javascript/javascript1/week2/homework02.js

```diff
@@ -0,0 +1,78 @@
+//Exercise 1 
+function getFullName(firstname, surname, useFormalName){
+    if(useFormalName){
+        return 'Lord' +  ' ' + firstname + ' ' + surname;
+    }
+    else{
+        return firstname + ' ' + surname ;
+    }
+}
+const fullname1 = (getFullName('Heema','Negandhi', true));
+const fullname2 = (getFullName('Karolina', 'Urnieziute'));
+console.log(fullname1);
+console.log(fullname2);
+
+// Exercise 2
+const daysInWeek =['sunday','monday','tuesday','wednesday','thursday','friday','saturday'];
+function getEventWeekday(number){
+    let d = new Date();
+    let n = d.getDay();
+    let i = (n + number) % 7;
+    return daysInWeek[i];
+}
+console.log(getEventWeekday(1));
+
+// Exercise 3 
+function youCreateThisFunctionName(tempearture){
```

Comment:
Hmm. If you were revisit this code 5 years later and saw a function named like this, you'd have no idea what the function does. It's always good to appropriately name functions based on what they do.

Reply:
surely made the changes.